integrationtests/mayavi/common.py

```diff
@@ -316,6 +316,9 @@ class TestCase(Mayavi):
     # `self.compare_image` was called in the test..
     offscreen = Bool(False)
 
+    # Profile for memory usage.
+    profile = Bool(False)
```

Comment:
What is a good name you suggest? I can think of `check_mem_usage`. 

Any references to explain why `profile` is misleading? I am only a novice when tackling memory issues :-).